src/Sarif.Viewer.VisualStudio.UnitTests/CodeAnalysisResultManagerTests.cs

```diff
@@ -276,6 +276,39 @@ public void CodeAnalysisResultManager_GetRebaselinedFileName_RelativeUri_Without
             remappedPathPrefixes[0].Item2.Should().Be(@"D:\Users\John\source\sarif-sdk");
         }
 
+        [Fact]
+        public void CodeAnalysisResultManager_GetRebaselinedFileName_ConstructPathFromSolutionPathRemapping()
+        {
+            const int RunId = 1;
+            const string ResultArtifactPath1 = @"Sarif/Notes.cs";
+            const string ResultArtifactPath2 = @"Sarif/OffsetInfo.cs";
+            const string SolutionPath = @"D:\Users\John\source\sarif-sdk\src";
+            const string ExpectedResolvedPath1 = SolutionPath + @"\Sarif\Notes.cs";
+            const string ExpectedResolvedPath2 = SolutionPath + @"\Sarif\OffsetInfo.cs";
```

Comment:
Path.Combine would be preferable to concatenation.

Reply:
fixed